sdk/include/reactos/wine/port.h

```diff
@@ -91,6 +91,7 @@ struct statfs;
 # endif /* defined(__BEOS__) */
 #endif /* !defined(HAVE_STATFS) */
 
+struct stat;
```

Comment:
For which function(s)/... is this needed?

Reply:
`sdk/include/reactos/wine/port.h:265:41: error: 'struct stat' declared inside parameter list will not be visible outside of this definition or declaration [-Werror]`